_data/simple-icons.json

```diff
@@ -13438,6 +13438,11 @@
             "hex": "8B8B8B",
             "source": "https://commons.wikipedia.org/wiki/File:WiiU.svg"
         },
+        {
+            "title": "Wiki.gg",
```

Comment:
```suggestion
            "title": "wiki.gg",
```

Their brand name is in lowercase.

Reply:
<img width="379" alt="image" src="https://github.com/simple-icons/simple-icons/assets/8186898/f5fe4584-88cf-48b0-adf0-30e81d0b043e">

@alistair3149 The shape is a perfect fit. But the icon name should be corrected.